camayoc/constants.py

```diff
@@ -329,6 +329,12 @@
 QCS_SCAN_PATH = 'scans/'
 """The path to the scans endpoint for CRUD tasks."""
 
+QCS_SCAN_STATES = ['completed', 'failed', 'paused', 'canceled', 'running']
+"""All the states that a quipucords scan can take."""
+
+QCS_SCAN_TERMINAL_STATES = ['completed', 'failed', 'paused', 'canceled']
```

Comment:
Instead of repeating I would define both states vars as:

```python
QCS_SCAN_TERMINAL_STATES = ('completed', 'failed', 'paused', 'canceled')
"""Scans to not change from these states without intervention."""

QCS_SCAN_STATES = QCS_SCAN_TERMINAL_STATES + ('running',)
"""All the states that a quipucords scan can take."""
```

I would also use tuples to make sure they are immutable